update/ansible/playbook/tasks/roles/initialization/tasks/main.yml

```diff
@@ -51,85 +55,27 @@
     mode: 0644
   when: docker_upgrade
 
-- name: Clean yum metadata
-  command: yum clean metadata
-  become: true
-  changed_when: True
-
-- name: Update (both)
-  block:
-    # This will implicitly create /srv/clickhouse
-    - name: Create clickhouse data directory
-      file:
-        path: "/srv/clickhouse/flags"
-        state: directory
-        owner: root
-        group: pmm
-        recurse: true
-
-    - name: Create empty file to convert clickhouse databases from ordinary to atomic
-      file:
-        path: "/srv/clickhouse/flags/convert_ordinary_to_atomic"
-        state: touch
-        owner: root
-        group: pmm
-
-    - name: Upgrade dashboards
-      include_role:
-        name: dashboards_upgrade
-  when: not pmm_current_version is version(pmm_image_version, '>=')
-
-- name: Create backup directory
-  file:
-    path: /srv/backup
-    state: directory
-
-- name: Create grafana DB
-  block:
-    - name: Create grafana database in postgres
-      postgresql_db:
-        name: grafana
-        state: present
-
-    - name: Create grafana user in postgres
-      postgresql_user:
-        db: grafana
-        name: grafana
-        password: grafana
-        priv: 'ALL'
-        expires: infinity
-        state: present
-      when: not ansible_check_mode
-  when: lookup('env','GF_DATABASE_URL') == '' and lookup('env','GF_DATABASE_HOST') == ''
-
-- name: Upgrade grafana database (Get the latest schema)
-  command: grafana cli --homepath=/usr/share/grafana admin data-migration encrypt-datasource-passwords
-  changed_when: True
-  when: lookup('env','PMM_TEST_HA_BOOTSTRAP') != '' and not pmm_current_version is version(pmm_image_version, '>=')
-
-- name: Create a working directory for VictoriaMetrics
-  file:
-    path: /srv/victoriametrics/data 
-    state: directory 
-    owner: pmm 
-    group: pmm
+- name: Upgrade dashboards
+  include_role:
+    name: dashboards_upgrade
+  when: need_upgrade
 
-- name: Create an empty configuration file for VictoriaMetrics
-  file: 
-    path: /etc/victoriametrics-promscrape.yml 
-    state: touch 
-    owner: pmm 
-    group: pmm
+- name: Upgrade clickhouse
+  include_role:
+    name: clickhouse
+  when: need_upgrade
```

Comment:
upgrade clickhouse should be done only during build period

Reply:
Let's rename job name then, maybe `clickhouse migration`?

Reply:
Done.